ReactApp/jsconfig.json

```diff
@@ -1,6 +1,10 @@
 {
   "compilerOptions": {
-    "baseUrl": "src"
-  },
-  "include": ["src"]
+    "baseUrl": ".",
```

Comment:
Why have we moved baseUrl (base directory used for resolving non-relative module names) to "."? I thought that "paths" will take care of "React-Automation-Studio".

Reply:
the entries here only affect vscode import suggestions, but basurl has no effect anymore because of the paths entry